drivers/sensor/hp206c/hp206c.h

```diff
@@ -72,8 +72,11 @@
 #endif
 /* end of default settings */
 
+struct hp206c_device_config {
+	const struct i2c_dt_spec i2c;
```

Comment:
```suggestion
	struct i2c_dt_spec i2c;
```

no need to make this const, as this is a struct, not a pointer. configuration constness comes from the definition of the config struct itself.